src/components/user/user.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './user.scss';
+
+export type UserType = {
```

Comment:
Rename to `Props` or `UserProps`.  Don't  export this props. If you need Type User in other files, move it to special file and export in user.tsx:

``` tsx
import { UserType } from 'types.ts';

type UserProps = {
   user: UserType;
}
```